CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Go Buildpack Changelog
 
 ## Unreleased
+* Add go1.14.14, use for go1.14
+* Add go1.15.6, use for go1.15
```

Comment:
@danp Whilst creating the changelog post, I noticed that the changelog here says `go1.15.6` when it should be `go1.15.7`, which I missed during review. I don't suppose you'd mind opening a PR to update it here? :-)

Reply:
Oops, good catch! https://github.com/heroku/heroku-buildpack-go/pull/443